source/fe/fe_values.cc

```diff
@@ -2674,7 +2676,8 @@ namespace internal
     AssertDimension(values.size(), n_quadrature_points);
 
     // initialize with zero
-    std::fill_n (values.begin(), n_quadrature_points, dealii::internal::NumberType<typename std::decay<Number>::type>::value(0.0));
+    std::fill_n (values.begin(), n_quadrature_points,
+                 dealii::internal::NumberType<typename std::decay<Number>::type>::value(0.0));
```

Comment:
I'm guessing so, but I could only confirm this at a later. I would need to either write a dedicated test (happy to do so) or I would have to propagate this change to one of my other development branches. Either way, I think that this is a defensive implementation.